test/test_prototype_transforms.py

```diff
@@ -1899,3 +1899,22 @@ def test_transpose_dimensions(dims):
             assert type(transformed_value) == torch.Tensor
         else:
             assert transformed_value is value
+
+
+class TestUniformTemporalSubsample:
+    @pytest.mark.parametrize(
+        "inpt",
+        [
+            torch.zeros(10, 3, 8, 8),
+            torch.zeros(1, 10, 3, 8, 8),
+            features.Video(torch.zeros(1, 10, 3, 8, 8)),
+        ],
+    )
+    def test__transform(self, inpt):
+        num_samples = 5
+        transform = transforms.UniformTemporalSubsample(num_samples)
+
+        output = transform(inpt)
+        assert type(output) is type(inpt)
```

Comment:
This was not strict enough before. You did `assert isinstance(output, type(inpt))`. This would also pass if the output is a `features.Video` in case the input was a `torch.Tensor`. This is not that far fetched of a mistake in the transform and thus I made the check more strict. With the new check, we assert that the type is retained.